src/environments/atari.jl

```diff
@@ -1,22 +1,5 @@
-module AtariWrapper
-
-using ArcadeLearningEnvironment, GR, Random
-using ReinforcementLearningBase
-
-export AtariEnv
-
-mutable struct AtariEnv{IsGrayScale,TerminalOnLifeLoss,N,S<:AbstractRNG} <: AbstractEnv
-    ale::Ptr{Nothing}
-    screens::Tuple{Array{UInt8,N},Array{UInt8,N}}  # for max-pooling
-    actions::Vector{Int64}
-    action_space::DiscreteSpace{UnitRange{Int}}
-    observation_space::MultiDiscreteSpace{Array{UInt8,N}}
-    noopmax::Int
-    frame_skip::Int
-    reward::Float32
-    lives::Int
-    seed::S
-end
+using .ArcadeLearningEnvironment
```

Comment:
Is the `.` required here? (Just wondering if I missed some basic knowledge here 😄 )

Reply:
That's specific for `Requires.jl`. In the readme it says
> In the @require block, or any included files, you can use or import the package, but note that you must use the syntax using .Gadfly or import .Gadfly, rather than the usual syntax. Otherwise you will get a warning about Gadfly not being in dependencies.

Reply:
Thanks, I totally missed it.